script.js

```diff
@@ -1,4 +1,84 @@
+var winCounts = 0;
+var playCounts = 0;
+var diceCounts = 0;
+
+var state = "begin";
+var numberofplays = "";
+var player1NumberList = [];
+var compNumberList = [];
+var numberOfRemainingPlays = "";
+
+var diceRoll = function () {
+  var randomDecimal = Math.random() * 6;
+  var ranNumber = Math.ceil(randomDecimal);
+  return ranNumber;
+};
+
+var valueFormed = function (firstDigit, secondDigit) {
```

Comment:
Remember that function names should start with a verb. A better name would be: generatePlayerScore